### PR TITLE
update GalaxyShedTool tosca type

### PR DESCRIPTION
**Pull request by mtangaro**
Update GalaxyShedTool type.
custom_types: updat GalaxyShedTool type to match indigo-dc.galaxy-tools role
examples/galaxy_tosca.yaml: uptade tosca to install tools
artifact/galaxy/galaxy_tool_configure.yml: chanded {{galaxy_instance_url}} var to {{instance_public_ip}}

### CODE REVIEW

File: examples/galaxy_tosca.yaml [hunk 1]
```diff
@@ -55,6 +59,14 @@ topology_template:
       requirements:
         - lrms: local_lrms
 
+    galaxy_tools:
+      type: tosca.nodes.indigo.GalaxyShedTool
+      properties:
+        galaxy_flavor: { get_input: flavor }
```
Comment (micafer) [inline examples/galaxy_tosca.yaml]: @mtangaro this is incorrect it must be only flavor.

File: examples/galaxy_tosca.yaml [hunk 2]
```diff
@@ -55,6 +59,14 @@ topology_template:
...
+        admin_api_key: { get_input: admin_api_key }
```
Comment (micafer) [inline examples/galaxy_tosca.yaml]: @mtangaro The nodetype tosca.nodes.indigo.GalaxyShedTool does not have this property you have to add it.

**Comment by alpegon** [comment]
add to whitelist

File: custom_types.yaml [hunk 3]
```diff
@@ -361,11 +359,9 @@ node_types:
         create:
           implementation: https://raw.githubusercontent.com/indigo-dc/tosca-types/master/artifacts/galaxy/galaxy_tools_configure.yml
           inputs:
-            galaxy_install_path: { get_property: [ HOST, install_path ] }
+            galaxy_flavor: { get_property: [ HOST, flavor ] }
```
Comment (micafer) [inline custom_types.yaml]: Hi @mtangaro,
Tests have passed OK. But I thinks that this line is not correct.
It should be:
galaxy_flavor: { get_property: [ **SELF**, flavor ] }
As you are refering the a property of this node, not a node of the GalaxyPortal node where it is hosted.

**Pull request merged**